docs/source/essentials/errors.mdx

```diff
@@ -137,3 +142,16 @@ For an example, it is possible for a person to not have a starship:
 ```
 
 In that case, `starship` is a true null and not an error.
+
+## Truth table
+
+| exception | data    | errors  | meaning                                                                     |
+|-----------|---------|---------|-----------------------------------------------------------------------------|
+| `null`    | `null`  | `null`  | Non-compliant server, if `data` is null, there should be at least one error |
```

Comment:
Nit: I'd use "Something" or "Not null" to make it a bit more legible (maybe it's just me, feel free to ignore!)

Reply:
Yup doesn't look good in preview. I'll add "not null" (or is it "non null" 🤔?)

![Screenshot 2024-08-06 at 16 19 40](https://github.com/user-attachments/assets/8c627650-8a90-4a7d-a753-6bc997be72f6)

Reply:
I went with `present`

![Screenshot 2024-08-06 at 16 19 40](https://github.com/user-attachments/assets/e3657ed8-b843-4022-9c35-259db1e88f16)